frontend/javascripts/admin/statistic/time_tracking_detail_view.tsx

```diff
@@ -0,0 +1,108 @@
+import { useFetch } from "libs/react_helpers";
+import React from "react";
+import { AnnotationTypeFilterEnum } from "./project_and_annotation_type_dropdown";
+import { getTimeTrackingForUserSummedPerAnnotation } from "admin/admin_rest_api";
+import dayjs from "dayjs";
+import { Col, Divider, Row } from "antd";
+import { formatMilliseconds } from "libs/format_utils";
+import _ from "lodash";
+import { APITimeTrackingPerAnnotation } from "types/api_flow_types";
+import { AnnotationStats } from "oxalis/view/right-border-tabs/dataset_info_tab_view";
+import { aggregateStatsForAllLayers } from "oxalis/model/accessors/annotation_accessor";
+
+type TimeTrackingDetailViewProps = {
+  userId: string;
+  dateRange: [number, number];
+  annotationType: AnnotationTypeFilterEnum;
+  projectIds: string[];
+};
+
+const ANNOTATION_OR_TASK_NAME_SPAN = 12;
+const STATISTICS_SPAN = 8;
+const TIMESPAN_SPAN = 4;
+
+const STYLING_CLASS_NAME = "time-tracking-details";
+
+const renderRow = (
+  userDataPerAnnotation: APITimeTrackingPerAnnotation[],
+): [Array<JSX.Element>, Array<JSX.Element>] => {
+  if (userDataPerAnnotation == null) return [[], []];
+  const groupedByProject = _.groupBy(userDataPerAnnotation, "projectName");
+  let taskRows: Array<JSX.Element> = [];
+  let annotationRows: Array<JSX.Element> = [];
+  for (const [project, loggedTimes] of Object.entries(groupedByProject)) {
+    if (project === "null") {
+      // explorative annotations
+      const tableRows = loggedTimes.map((timeEntry) => (
+        <Row>
+          <Col span={ANNOTATION_OR_TASK_NAME_SPAN}>
+            <a href={`annotations/${timeEntry.annotation}`}>Annotation: {timeEntry.annotation} </a>
+          </Col>
+          <Col span={STATISTICS_SPAN}>
+            <AnnotationStats
+              stats={aggregateStatsForAllLayers(timeEntry.annotationLayerStats)}
+              asInfoBlock={false}
+            />
+          </Col>
+          <Col span={TIMESPAN_SPAN}>{formatMilliseconds(timeEntry.timeMillis)}</Col>
+        </Row>
+      ));
+      annotationRows = annotationRows.concat(tableRows);
+    } else {
+      // tasks
+      taskRows.push(
+        <Row style={{ fontWeight: "bold", margin: 20 }}>
+          <Col>{project}</Col>
+        </Row>,
+      );
+      const tableRows = loggedTimes.map((timeEntry) => (
+        <Row style={{ lineHeight: 3.3 }}>
```

Comment:
why does this row have a special lineHeight?

Reply:
because there are fewer segments stats, the lines would be much lower, and I thought it looked weird (in the picture I removed the styling). Will add a comment :)

Reply:
![image](https://github.com/scalableminds/webknossos/assets/57364771/f894a52a-9da8-453b-9004-dd42bb734df9)

Reply:
wait, I am not sure this isn't due to the way I am testing

Reply:
ah, I think the segment count is never shown for tasks because its not possible to create segments in tasks, right?

Reply:
depending on the settings in the task type selected during task creation, it should be possible to create segments in tasks. But it may well be that the sampleTaskType in the development setup is set to skeleton-only.
![image](https://github.com/scalableminds/webknossos/assets/1390035/958a1e82-f520-40d4-94fb-c9741a5e9bb0)

Reply:
alright, I will test the styling with skeleton and volume then :)

Reply:
yeah, I will need to update this 
![image](https://github.com/scalableminds/webknossos/assets/57364771/0afd1b33-ded7-4df1-ad5a-4aebb614c0c5)

Reply:
I adjusted the styling so that it seems to look okay in both cases :)